client/src/pages/CreateBlogPage.jsx

```diff
@@ -29,125 +28,181 @@ const CreateBlogPage = () => {
-                    fill="none"
-                    viewBox="0 0 24 24"
+                  <path
                     stroke="currentColor"
-                  >
-                    <path
-                      strokeLinecap="round"
-                      strokeLinejoin="round"
-                      strokeWidth="2"
-                      d="M6 18L18 6M6 6l12 12"
-                    />
-                  </svg>
-                </button>
+                    strokeLinecap="round"
+                    strokeLinejoin="round"
+                    strokeWidth="2"
+                    d="M13 13h3a3 3 0 0 0 0-6h-.025A5.56 5.56 0 0 0 16 6.5 5.5 0 0 0 5.207 5.021C5.137 5.017 5.071 5 5 5a4 4 0 0 0 0 8h2.167M10 15V6m0 0L8 8m2-2 2 2"
+                  />
+                </svg>
+                <p className="mb-2 text-sm text-gray-500">
+                  <span className="font-semibold">Click to upload</span> or drag
+                  and drop
+                </p>
+                <p className="text-xs text-gray-500">PNG or JPG</p>
               </div>
-            )}
-            {!coverImageUrl && (
-              <label htmlFor="dropzone-file" className="flex flex-col items-center justify-center w-full h-96 border-2 border-gray-300 border-dashed rounded-lg cursor-pointer bg-gray-50 hover:bg-gray-100">
-                <div className="flex flex-col items-center justify-center pt-5 pb-6">
-                  <svg className="w-8 h-8 mb-4 text-gray-500" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 20 16">
-                    <path stroke="currentColor" strokeLinecap="round" strokeLinejoin="round" strokeWidth="2" d="M13 13h3a3 3 0 0 0 0-6h-.025A5.56 5.56 0 0 0 16 6.5 5.5 0 0 0 5.207 5.021C5.137 5.017 5.071 5 5 5a4 4 0 0 0 0 8h2.167M10 15V6m0 0L8 8m2-2 2 2" />
-                  </svg>
-                  <p className="mb-2 text-sm text-gray-500"><span className="font-semibold">Click to upload</span> or drag and drop</p>
-                  <p className="text-xs text-gray-500">PNG or JPG</p>
-                </div>
-                <input id="dropzone-file" type="file" className="hidden" onChange={handleCoverImageChange} />
-              </label>
-            )}
-          </div>
-
-          <div className='mt-3 mb-10'>
-            <textarea
-              id="summary"
-              placeholder='Type your blog summary here...'
-              value={summary}
-              onChange={handleSummaryChange}
-              rows={4}
-              className='w-full italic::placeholder border rounded'
-              style={{
-                fontSize: '1rem',
-                padding: '10px',
-                resize: 'none',
-                height: '120px',
-              }}
-            />
-          </div>
+              <input
+                id="dropzone-file"
+                type="file"
+                className="hidden"
+                onChange={handleCoverImageChange}
+              />
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Missing file type validation on upload</b></div>
<div id="fix">

Consider adding file type validation to the `input` element to restrict uploads to only PNG and JPG files. Could add `accept="image/png,image/jpeg"` attribute.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<blockquote>Check the AI-generated fix before applying</blockquote>
<div id="code">


```suggestion
              <input
                id="dropzone-file"
                type="file"
                accept="image/png,image/jpeg"
                className="hidden"
                onChange={handleCoverImageChange}
              />
```

</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/Divyanshu9822/blog-mern-app/pull/22#issuecomment-2544587479>#245782</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged